CONTRIBUTING.rst

```diff
@@ -0,0 +1,160 @@
+
+**********************
+The Developer Workflow
+**********************
+
+General Notes
+=============
+
+* The terminology we use is based on the `Integrator Workflow
+  <http://en.wikipedia.org/wiki/Integrator_workflow>`_
+
+* Use a branching workflow similar to the one described at
+  http://progit.org/book/ch3-4.html.
+
+* Keep your own "master" and "develop" branches in sync with the blessed
+  repository's "master" and "develop" branches. Specifically, do not push your
+  own commits directly to your "master" and "develop" branches.
+
+* Any commit should *pass all tests* (see `Running Tests`_).
+
+* See the `An Example`_ section below for a full walk through
+
+Issuing a Pull Request
```

Comment:
I think that 'acquiring cyclus' and the developer workflow should be above the 'issuing and reviewing a pull request' sections here (and perhaps that means the cautions should follow the pull request-related text to the bottom as well)